seed/views/v3/portfolio_manager.py

```diff
@@ -278,21 +282,24 @@ def get_list_of_report_templates(self):
             self.login_and_set_cookie_header()
 
         # get the report data
-        url = 'https://portfoliomanager.energystar.gov/pm/reports/reportData'
         try:
-            response = requests.get(url, headers=self.authenticated_headers)
+            json_authenticated_headers = self.authenticated_headers.copy()
+            json_authenticated_headers['Accept'] = 'application/json'
+            json_authenticated_headers['Content-Type'] = 'application/json'
+            response = requests.get(self.REPORT_URL, headers=json_authenticated_headers)
 
         except requests.exceptions.SSLError:
             raise PMExcept('SSL Error in Portfolio Manager Query; check VPN/Network/Proxy.')
         if not response.status_code == status.HTTP_200_OK:
             raise PMExcept('Unsuccessful response from report template rows query; aborting.')
 
-        template_object = self.parse_template_response(response.text)
```

Comment:
I think we can remove `parse_template_response` now

Reply:
great catch. removed.